template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApplication-CSharp/Program.cs

```diff
@@ -1,2 +1,21 @@
-﻿Console.WriteLine("Hello, World!");
+﻿#if (csharpFeature_TopLevelProgram)
 // See https://aka.ms/new-console-template for more information
+#endif
+#if (!csharpFeature_ImplicitUsings)
+using System;
+
+#endif
+#if (!csharpFeature_TopLevelProgram)
+namespace Company.ConsoleApplication1
```

Comment:
Are you planning on duplicating this content for the file-scoped namespaces feature?

Reply:
I haven't tested it yet, but I guess no. Top-level programs are supported since 9.0 and file scoped namespaces are supported since 10.0, so there should be no need as if top level program is not supported, file scoped namespaces won't be supported too.

Reply:
ah of course, this is `Program.cs` :smile: 👍